website/docs/reference/resource-configs/grants.md

```diff
@@ -98,32 +113,109 @@ See [configs and properties](configs-and-properties) for details.
 </TabItem>
 </Tabs>
 
-## Definition
+### Grant config inheritance
 
-You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt.
+If `grants` are set for the same model in multiple places, such as in `dbt_project.yml` and in a more-specific `.sql` or `.yml` file, the default behavior is to "merge and clobber" each privilege. That is, dbt will replace the less-specific set of grantees with the more-specific set of grantees.
 
-## Database-specific requirements and notes
+For example:
 
-While we try to standardize the terms we use to describe different features, you will always find nuances in different databases. This section outlines some of those database-specific requirements and notes.
+<File name='dbt_project.yml'>
 
-### Common syntax 
+```yml
+models:
+  +grants:
+    select: ['user_a', 'user_b']
+```
+
+</File>
+
+<File name='models/specific_model.sql'>
+
+```sql
+{{ config(grants = {'select': ['user_c']}) }}
+```
+
+</File>
+
+As a result of this configuration, `specific_model` will grant the `select` privilege to `user_c` ONLY.
+
+Let's say we wanted to _add_ `user_c` to the existing list of grantees receiving the `select` privilege on `specific_model`, rather than _replacing_ that list entirely. To accomplish that, we can use the `+` ("addition") symbol, prefixing the name of the privilege:
+
+<File name='models/specific_model.sql'>
+
+```sql
+{{ config(grants = {'+select': ['user_c']}) }}
```

Comment:
so if there's a single grant config given at the top-level of the project, every other grant statement in the project needs to have the `+` prefix, else it will remove the top-level grant.

Reply:
Correct. Thinking here is, better safe than sorry, really — accidentally too-few grants is preferable to accidentally too many. It's an opt-in to say, "All of the existing ones, and one more too!"